### PR TITLE
changes for argus brand

### PR DESCRIPTION
**Pull request by warrenstephens**

### CODE REVIEW

**Review by muralikarthikk — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -55,7 +55,7 @@ start:
 	if [ "${CREATE_TEST_USERS}" = "true" ]; then ${MANAGE} shell < bin/createtestusers.py; fi
 	${MANAGE} collectstatic --clear --no-input
 	${MANAGE} spectacular --file ./appstore/schema.yml
-	#bash /usr/src/inst-mgmt/bin/populate_env.sh /usr/src/inst-mgmt/appstore/static/frontend/env.json
+	bash /usr/src/inst-mgmt/bin/populate_env.sh /usr/src/inst-mgmt/appstore/static/frontend/env.json
```
Comment (ismagilani) [inline Makefile:58]: why did we uncomment this line? @warrenstephens @muralikarthikk

**Review by ismagilani — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (warrenstephens) [reply Makefile:58]: My understanding is that it needed to be commented out in order to run locally -- and the commented out version slipped into the original PR by mistake.

Comment (ismagilani) [reply Makefile:58]: So should it be reverted to being commented out if we are pushing to develop?

Comment (muralikarthikk) [reply Makefile:58]: I think this line is to make some variables(like enable/disable search, workspaces etc.) available to the React frontend passed to the container using the environment variables. This line is necessary though. Bo might have a better understanding about this.

**Pull request merged**